packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/RancherDetail/RancherDetail.component.tsx

```diff
@@ -118,7 +120,7 @@ const RancherDetail = ({
     : null;
 
   const onAccessRancherUrl = () =>
-    trackAction(TrackingPageView.DetailRancher, TrackingEvent.accessUi);
+    trackClick({ actions: [TrackingEvent.accessUi] });
```

Comment:
if url is inside the app, you should maybe also use `navigate(url)` to avoid reloading the app